ext/standard/array.c

```diff
@@ -4061,6 +4094,56 @@ PHP_FUNCTION(array_values)
 }
 /* }}} */
 
+/* {{{ array_value_result_helper
+ * Get a value out of a HashTable at the given position and copy the value into return_value.
+ */
+static inline
+void array_value_result_helper(HashTable *ht, HashPosition *pos, zval *return_value)
```

Comment:
This should not split over two lines.

Reply:
This seems to be inconsistently but I'll gladly fix this to an oneliner